src/app/components/TranscriptExperiment/MediaIndicator/index.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
+import { mediaIcons } from '#psammead/psammead-assets/src/svgs';
+import { ReactElement } from 'react';
+import style from './index.styles';
+
+type MediaIndicatorProps = {
+  datetime?: string;
+  duration?: string;
+  durationSpoken?: string;
+  type?: string;
+  title?: string;
+  guidanceMessage?: string | null;
+};
+
+const MediaIndicator = ({
+  datetime,
+  duration,
+  durationSpoken,
+  type = 'Video',
+  title = '',
+}: MediaIndicatorProps) => {
+  const hiddenText = `${type}, ${
+    datetime && duration && durationSpoken
+      ? `"${title}", ${durationSpoken}`
+      : `"${title}"`
+  } `.trim();
+
+  const validDuration = datetime && duration && durationSpoken;
```

Comment:
Improve readability
```suggestion
  const hasDuration = datetime && duration && durationSpoken;
```

or 
```suggestion
  const shouldDisplayDuration = datetime && duration && durationSpoken;
```